src/expanses/expanses.controller.ts

```diff
@@ -0,0 +1,98 @@
+import ExpanseNotFoundException from "../exceptions/ExpanseNotFoundException";
+import * as express from "express";
+import validationMiddleware from "../middleware/validation.middleware";
+import CreateExpanseDto from "./expanses.dto";
+import Expanse from "./expanses.interface";
+import DbInstance from "db/db.instance";
+
+class ExpansesController {
+  public path = "/expanses";
+  public router = express.Router();
+
+  private expanses;
+
+  constructor() {
+    this.initializeRoutes();
+    this.expanses = new DbInstance();
+  }
```

Comment:
Let's create a basic (abstract) class `Controller` (you can add `initializeRoutes` method to it as required)

<img width="290" alt="image" src="https://user-images.githubusercontent.com/86183007/203373374-80aa0c65-3e26-44bc-916e-c9688cb6ccaf.png">